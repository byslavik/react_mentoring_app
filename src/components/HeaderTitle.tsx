import * as React from 'react';

import * as s from './scss/Header.scss';

export class HeaderTitle extends React.Component {

    render() {
        return (<h1 className={s.siteHeader}>Find your movie</h1>)
    }
}
