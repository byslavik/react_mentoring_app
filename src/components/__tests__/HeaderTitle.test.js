import * as React from 'react';
import {HeaderTitle} from '../HeaderTitle';
import * as renderer from 'react-test-renderer';


describe('HeaderTitle component', ()=>{
  it('HeaderTitle component shows correctly', () => {
    const tree = renderer.create(
      <HeaderTitle />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
})