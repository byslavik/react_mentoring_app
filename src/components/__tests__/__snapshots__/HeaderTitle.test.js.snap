// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderTitle component HeaderTitle component shows correctly 1`] = `
<h1
  className={undefined}
>
  Find your movie
</h1>
`;
